src/VisualStudio/IntegrationTest/New.IntegrationTests/InProcess/ShellInProcess.cs

```diff
@@ -2,20 +2,71 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System;
 using System.Linq;
 using System.Reflection;
 using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.CodeAnalysis.UnitTests;
+using Microsoft.VisualStudio.Editor;
 using Microsoft.VisualStudio.Shell;
 using Microsoft.VisualStudio.Shell.Interop;
 using Microsoft.VisualStudio.Threading;
 using IAsyncDisposable = System.IAsyncDisposable;
+using IOleCommandTarget = Microsoft.VisualStudio.OLE.Interop.IOleCommandTarget;
+using OLECMDEXECOPT = Microsoft.VisualStudio.OLE.Interop.OLECMDEXECOPT;
 
 namespace Microsoft.VisualStudio.Extensibility.Testing
 {
     internal partial class ShellInProcess
     {
+        public async Task<(Guid commandGroup, uint commandId)> PrepareCommandAsync(string command, CancellationToken cancellationToken)
```

Comment:
📝 This was implemented over in https://github.com/microsoft/vs-extension-testing/pull/104 so I can remove them as part of a harness update.

Reply:
➡️ Now updated to use harness features